ngraph/core/include/openvino/core/descriptor/input.hpp

```diff
@@ -95,7 +95,7 @@ class OPENVINO_API Input {
     const ngraph::Shape& get_shape() const;
 
     /// \return the partial shape of the connected output
-    const PartialShape& get_partial_shape() const;
+    const Shape& get_partial_shape() const;
```

Comment:
what will we do with such names? Do we have a ticket for it?

Can we depending on whether we have new / old API have different names using `ifdef`?

Reply:
Yes, we have a ticket for it.